sdcm/logcollector.py

```diff
@@ -413,7 +414,8 @@ def get_grafana_screenshot(self, node, local_dst):
                                                                     datetime.datetime.now().strftime("%Y%m%d_%H%M%S"),
                                                                     node.name))
                 LOGGER.debug("Get screenshot for url %s, save to %s", grafana_url, screenshot_path)
-                remote_browser.get_screenshot(grafana_url, screenshot_path, screenshot['resolution'])
+                remote_browser.get_screenshot(grafana_url, screenshot_path,
+                                              screenshot['resolution'], self.wait_scr_timeout)
```

Comment:
please use `load_page_screenshot_delay` keyword argument for the call to `get_screenshot()`

also I was under the assumption we had some more clever way to wait for the whole dashboard to be loaded before we take the screenshot...

just sleeping 2mins seems a bit excessive, and not exactly a timeout

Reply:
@aleksbykov please improve naming as Israel suggested

Reply:
2 mins is not excessive.
I prefer we will wait 2 mins or even more than fighting with empty screenshots.

Reply:
Just saying that tomorrow it would might need 3min... and so forth.

Reply:
@fruch @bentsi fixed naming.
also revert taking screenshot, which looks was lost during migrating to docker based mixin